lib/DataQueue/src/DataQueue.cpp

```diff
@@ -18,12 +18,12 @@ void DataQueue::add(String id, float value) {
 
 void DataQueue::publish(String event, PublishFlags flag1, PublishFlags flag2) {
 	String payload = jsonMaker->get();
-	publishQueue->publish(event, payload, flag1, flag2);
+	//publishQueue->publish(event, payload, flag1, flag2);
 	jsonMaker->refresh();
 }
 
 void DataQueue::init() {
-	publishQueue = new PublishQueueAsyncRetained(publishQueueRetainedBuffer, uint16_t(sizeof(publishQueueRetainedBuffer)));
+	//publishQueue = new PublishQueueAsyncRetained(publishQueueRetainedBuffer, uint16_t(sizeof(publishQueueRetainedBuffer)));
```

Comment:
> Not currently sure how the DataQueue library works, however it is dependent on another library that's not in the repo (or not created yet). Have commented out lines that wouldn't compile and we can come back to it later

Have you confirmed whether you have the submodule cloned? Unfortunately, submodules don't clone by default so if you don't initialize them, they'll be empty.

Reply:
Yes, that was the issue! Fixed now and I've uncommented these lines again